BoneLib/BoneLib/HeadNotificationData.cs

```diff
@@ -0,0 +1,20 @@
+﻿using UnityEngine;
+
+namespace BoneLib
+{
+    public struct HeadNotificationData
+    {
+        public string title;
+        public string subtitle;
+        
+        public AudioClip clip;
+        public float holdTime;
+        public bool isAvatarLevel;
+
+        public Sprite mainSprite;
+        public Sprite spriteA;
+        public Sprite spriteB;
+        public Sprite spriteC;
+        public Sprite spriteD;
+    }
```

Comment:
nit: with how many fields there are on here, and how many of them are reference fields, this should probably just be a class